docs/3.x.x/en/getting-started/installation.md

```diff
@@ -5,8 +5,8 @@ Installation is very easy and only takes a few seconds.
 ## Requirements
 
 Please make sure your computer/server meets the following requirements:
- - [Node.js](https://nodejs.org) >= 9: Node.js is a server platform which runs JavaScript. Installation guide [here](https://nodejs.org/en/download/).
```

Comment:
What's the problem with Node 9? I'm using it and I don't see any problem. When changed the support? 🤔

Reply:
Node 10 it is. See comment from @derrickmehaffy : https://github.com/strapi/strapi/issues/2023#issuecomment-424818662